.changes/config.json

```diff
@@ -144,9 +144,66 @@
       "manager": "javascript",
       "assets": [
         {
-          "path": "./tooling/api/tauri-apps-api-${ pkgFile.version }.tgz",
+          "path": "./tooling/api/dist/tauri-apps-api-${ pkgFile.version }.tgz",
           "name": "api-${ pkgFile.version }.tgz"
         }
+      ],
+      "prepublish": [
+        {
+          "command": "yarn",
+          "dryRunCommand": true
+        },
+        {
+          "command": "echo \"# Yarn Audit\n\"",
+          "dryRunCommand": true,
+          "pipe": true
+        },
+        {
+          "command": "echo \"<details>\n<summary>click to view</summary>\n\n\\`\\`\\`\"",
+          "dryRunCommand": true,
+          "pipe": true
+        },
+        {
+          "command": "yarn audit",
+          "dryRunCommand": true,
+          "runFromRoot": true,
+          "pipe": true
+        },
+        {
+          "command": "echo \"\\`\\`\\`\n\n</details>\n\"",
+          "dryRunCommand": true,
+          "pipe": true
+        },
+        {
+          "command": "yarn build",
+          "dryRunCommand": true
+        },
+        {
+          "command": "cd ./dist && npm pack",
```

Comment:
I just confirmed that this won't work. There was an issue upstream where the args aren't processed correctly so that is run as one single command. We need to `shell: true`, and I fixed it temporarily by searching for `||`, but I didn't consider `&&`. Can we move all of this into the `package.json` script?

Reply:
can I just add another command before it that changes the directory instead of combining them?